bundles/core/src/test/java/com/adobe/cq/commerce/core/components/client/MagentoGraphqlClientTest.java

```diff
@@ -61,6 +61,15 @@
     private static final ValueMap MOCK_CONFIGURATION = new ValueMapDecorator(ImmutableMap.of("cq:graphqlClient", "default", "magentoStore",
         "my-store"));
 
+    private static final List<Header> expectedHeaders = new ArrayList<>();
```

Comment:
Is there any reason why this needs to be a static member? Afaics this is local to the testCustomerHeaders test only.

Reply:
Yes, I'll move it there.